tmail-backend/combined-identity/src/main/java/com/linagora/tmail/combined/identity/CombinedUserDAO.java

```diff
@@ -4,33 +4,36 @@
 import java.util.Optional;
 
 import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
 
 import org.apache.james.core.Username;
 import org.apache.james.user.api.UsersRepositoryException;
 import org.apache.james.user.api.model.User;
-import org.apache.james.user.cassandra.CassandraUsersDAO;
 import org.apache.james.user.ldap.ReadOnlyLDAPUsersDAO;
 import org.apache.james.user.lib.UsersDAO;
 import org.reactivestreams.Publisher;
 import org.slf4j.LoggerFactory;
 
 public class CombinedUserDAO implements UsersDAO {
     public static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(CombinedUserDAO.class);
+    public static final String BACKPORT_INJECT_NAME = "backport";
```

Comment:
Can you explain the `backport` name?

Reply:
Before this commit
`CombinedUserDAO` = `CassandraUserDAO` + `LDAP`
With the change `CassandraUserDAO` to `UserDAO` (with injectName) it will help we can use CombiendUserDAO for Postgres.
"backport" is just another name for the difference from primary `UserDAO` (here is CombinedUserDAO)

Reply:
> "backport" is just another name for the difference from primary UserDAO (here is CombinedUserDAO)

Maybe a better name? (`backport` seems not reflect the context to me)
`persisted_users_dao` / `database_users_dao` / `local_users_dao`....

Reply:
`database_users_dao` looks ok

Reply:
Or `combined_users_dao`? simple :)